### PR TITLE
Support SSL / TLS connections

### PR DESCRIPTION
**Pull request by g-andrade**

### CODE REVIEW

**Comment by lrascao** [comment]
👍 after squashing

**Comment by g-andrade** [comment]
Squashed as much as I could without triggering conflicts. Is it OK now?

File: src/eredis_client.erl [hunk 1]
```diff
@@ -267,7 +276,6 @@ reply(Value, Queue) ->
 
 %% @doc Send `Value' to each client in queue. Only useful for sending
 %% an error message. Any in-progress reply data is ignored.
--spec reply_all(any(), eredis_queue()) -> ok.
```
Comment (paulo-ferraz-oliveira) [inline src/eredis_client.erl:270]: Why?

**Review by paulo-ferraz-oliveira — APPROVED** [review]
I approve this

**Pull request merged**